mypy/nodes.py

```diff
@@ -1031,9 +1031,18 @@ def serialize(self) -> JsonDict:
     @classmethod
     def deserialize(self, data: JsonDict) -> 'ClassDef':
         assert data['.class'] == 'ClassDef'
+
+        tvs_ = [mypy.types.deserialize_type(v) for v in data['type_vars']]
+        tvs = []
+        for tv in tvs_:
+            # mypy tells me to expand `tv`, which isn't necessary I believe?
+            assert isinstance(tv, mypy.types.TypeVarLikeType)  # type: ignore[misc]
+            tvs.append(tv)
```

Comment:
This seems overly complicated. What about using `cast` inside the list comprehension?
```py
[cast(mypy.types.TypeVarLikeType, mypy.types.deserialize_type(v)) for v in data['type_vars']]
```

Reply:
This is nice!